.github/workflows/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  rule_linter:
+    runs-on: ubuntu-latest
+    steps:
+    # We check the submodules separately as the rules submodule's reference may not be our PR/master
+    - name: Checkout capa without submodules
+      uses: actions/checkout@v2
+      with:
+        repository: fireeye/capa
+    - name: Checkout capa-rules
+      uses: actions/checkout@v2
+      with:
+        path: rules
+    - name: Checkout capa-rules
```

Comment:
This should be "Checkout capa-testfiles"?

Reply:
copy-paste... 🙈